apps/admin-portal/src/api/certificates.ts

```diff
@@ -18,14 +18,13 @@
 
 import { IdentityAppsApiException } from "@wso2is/core/exceptions";
 import { Certificate, HttpMethods } from "@wso2is/core/models";
-import { AxiosHttpClient, AxiosHttpClientInstance } from "@wso2is/http";
+import { OAuth } from "@wso2is/oauth-web-worker";
 import { store } from "../store";
 
 /**
  * Initialize an axios Http client.
- * @type { AxiosHttpClientInstance }
  */
-const httpClient: AxiosHttpClientInstance = AxiosHttpClient.getInstance();
+const httpClient = OAuth.getInstance().httpRequest;
```

Comment:
Is `httpRequest` a function in the instance returned by `OAuth` class?

Reply:
Yes

Reply:
(y)